include/leo_object_storage.hrl

```diff
@@ -228,12 +229,14 @@
                 ?AVS_HEADER_VSN_2_2 ->
                     term_to_binary({_AddrId, _Key});
                 ?AVS_HEADER_VSN_2_4 ->
+                    _Key;
+                ?AVS_HEADER_VSN_3_0 ->
                     _Key
             end
         end).
 
 -define(AVS_HEADER_VSN,     <<?AVS_HEADER_VSN_TOBE/binary,13,10>>).
--define(AVS_PART_OF_HEADER, <<"CHKSUM:128,KSIZE:16,BLEN_MSIZE:32,DSIZE:32,OFFSET:64,ADDRID:128,CLOCK:64,TIMESTAMP:42,DEL:1,BUF:437,CHUNK_SIZE:32,CHUNK_NUM:24,CHUNK_INDEX:24",13,10>>).
+-define(AVS_PART_OF_HEADER, <<"CHKSUM:128,KSIZE:16,BLEN_MSIZE:32,DSIZE:32,OFFSET:64,ADDRID:128,CLOCK:64,TIMESTAMP:42,DEL:1,BUF:437,CHUNK_SIZE:32,CHUNK_NUM:24,CHUNK_INDEX:24,SSEC_KEY_HASH:32,SSEC_IV",13,10>>).
```

Comment:
ditto. SSEC_KEY_HASH:32,SSeC_IV are no longer needed here?

Reply:
@mocchira Right, `SSEC_KEY_HASH:32,SSEC_IV` are not necessary. I'll rollback this line.